imports/api/tasks/tasks.js

```diff
@@ -1,3 +1,17 @@
 import { Mongo } from 'meteor/mongo';
 
 export const Tasks = new Mongo.Collection('tasks');
+
+var Schemas = {};
+
+Schemas.Tasks = new SimpleSchema({
+  name: { type: String },
+  completed_on: {type: Date, optional: true},
+  created_on: {type: Date},
+  due_on: { type: Date },
+  is_urgent: { type: Boolean},
+  is_important: { type: Boolean}
+
+});
+
+Tasks.attachSchema(Schemas.Tasks);
```

Comment:
need to fix this.
Remove 'export' from line 3. At the bottom of the file, type export Tasks